chainer/functions/connection/n_step_lstm.py

```diff
@@ -373,8 +374,10 @@ def _stack_weight(ws):
 
 
 def n_step_lstm(
-        n_layers, dropout_ratio, hx, cx, ws, bs, xs):
-    """Stacked Long Short-Term Memory function for sequence inputs.
+        n_layers, dropout_ratio, hx, cx, ws, bs, xs, **kwargs):
+    """n_step_lstm(n_layers, dropout_ratio, hx, cx, ws, bs, xs)
```

Comment:
`kwargs` is just for showing an error message when obsolete API is used. I think it is redundant to write in the document. How do you think?